core/src/main/resources/META-INF/native-image/com.datastax.oss/java-driver-core/reflection.json

```diff
@@ -0,0 +1,214 @@
+[
```

Comment:
Contents of this file largely taken from https://github.com/absurdfarce/graal-cassandra-driver/blob/master/reflection.json